dietpi/dietpi-software

```diff
@@ -886,6 +886,13 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_CATX[$software_id]=4
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/cloud/#vaultwarden'
 		aSOFTWARE_DEPS[$software_id]='16 87'
+		#------------------
+		software_id=192
+
+		aSOFTWARE_NAME[$software_id]='k3s'
+		aSOFTWARE_DESC[$software_id]='The certified Kubernetes distribution built for IoT & Edge computing'
+		aSOFTWARE_CATX[$software_id]=4
```

Comment:
I'd say it fits better into `'●─ System Stats & Management ' #8` next to Docker, doesn't it?

Reply:
Category was one of the things I had trouble deciding on. Didn't even realize docker was in that category, but yeah, that makes sense. I'll move it.